### PR TITLE
Hero

### PR DESCRIPTION
**Pull request by LiorGnieslawMoveo**

### CODE REVIEW

File: epicure-frontend/src/components/Hero/Hero.scss [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    justify-content: center;
+    align-items: center;
+
+    @include desktop {
+        height: 696px;
+    }
+
+    .hero-overlay {
+        display: flex;
+        background: #FFFFFFE0;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/components/Hero/Hero.scss:18]: set as variable

File: epicure-frontend/src/components/Hero/Hero.scss [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            @include desktop {
+                width: 404px;
+                padding: 36px 126px, 36px, 126px;
+            }
+            
+            .hero-text {
+                @include font-style;
+                font-size: 23.8px;
+                line-height: 32px;
+                letter-spacing: 1.9700000286102295px;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/components/Hero/Hero.scss]: round it

File: epicure-frontend/src/components/Hero/Hero.scss [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                    width: 505px;
+                    height: 48px;
+                    gap: 16px;
+                }
+
+                input {
+                    @include font-style;
+                    font-size: 12px;
+                    line-height: 15px;
+                    letter-spacing: 1.2899999618530273px;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/components/Hero/Hero.scss]: round it

File: epicure-frontend/src/components/Hero/Hero.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import SearchIcon from '../../assets/images/search-icon.svg'
+
+const Hero = () => {
+    return (
+        <>
+            <div className='hero-container' >
+                <div className='hero-overlay'>
+                    <div className='hero-content'>
+                        <div className='hero-text'>Epicure works with the top chef restaurants in Tel Aviv</div>
+                        <div className='search-bar'>
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/components/Hero/Hero.tsx:11]: set as input and not div

**Review by MoranOshiaMoveo — CHANGES_REQUESTED** [review]

File: epicure-frontend/src/components/Hero/Hero.scss (hunk 1 above)
Comment (LiorGnieslawMoveo) [reply epicure-frontend/src/components/Hero/Hero.scss:18]: I've created a variables file in the prior branches so once I will merge them I will apply the styling on all the components - is that ok?

File: epicure-frontend/src/components/Hero/Hero.tsx (hunk 4 above)
Comment (LiorGnieslawMoveo) [reply epicure-frontend/src/components/Hero/Hero.tsx:11]: I have in this div a button and an input, if I will change this the input wont be able to contain the button.

**Review by MoranOshiaMoveo — APPROVED** [review]

**Review by MoranOshiaMoveo — APPROVED** [review]

**Pull request merged**